lib/atomic_web/live/user_live/form_component.ex

```diff
@@ -5,7 +5,7 @@ defmodule AtomicWeb.UserLive.FormComponent do
 
   @impl true
   def update(%{user: user} = assigns, socket) do
-    changeset = Accounts.update_user(user)
+    changeset = Accounts.update_user(user, %{})
```

Comment:
```suggestion
    changeset = Accounts.update_user(user)
```
Make `%{}` be the default value.